engine/src/main/java/org/destinationsol/game/screens/TalkScreen.java

```diff
@@ -98,7 +98,9 @@ public void updateCustom(SolApplication solApplication, SolInputManager.InputPoi
         boolean sellShips = shipsControl.isJustOff();
         boolean hire = hireControl.isJustOff();
         if (sell || buy || sellShips || hire) {
-            inventoryScreen.setOperations(sell ? inventoryScreen.sellItems : buy ? inventoryScreen.buyItems : sellShips ? inventoryScreen.changeShip : inventoryScreen.hireShips);
+            inventoryScreen.setOperations(sell ? inventoryScreen.sellItems
+                    : buy ? inventoryScreen.buyItems
+                            : sellShips ? inventoryScreen.changeShip : inventoryScreen.hireShips);
```

Comment:
This would be much clearer as something like:
```
InventoryOperations inventoryOperations = inventoryScreen.hireShips;
if (sell) {
    inventoryOperations = inventoryScreen.sellItems;
} else if (buy) {
    inventoryOperations = inventoryScreen.buyItems;
} else if (sellShips) {
    inventoryOperations = inventoryScreen.changeShip;
}
inventoryScreen.setOperations(inventoryOperations);
```